lib/logux/logging.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Logux
+  module Logging
```

Comment:
Let's revert it back to Logger name. I don't see reasons to use `Logging`, instead of familiar `Logger`.